pkg/gui/controllers/helpers/window_arrangement_helper_test.go

```diff
@@ -12,6 +12,8 @@ import (
 	"golang.org/x/exp/slices"
 )
 
+const appStateRebasing = "Rebasing /"
```

Comment:
I put this here not where the type definition of `AppStatus` because this seems only for test.